docs/contract/dataplanes-dataframes.md

```diff
@@ -0,0 +1,91 @@
+
+# Grafana data structure
+
+Grafana supports a variety of different data sources, each with its own data model. To make this possible, Grafana consolidates the query results from each of these data sources into one unified data structure called a **data frame**. The **data plane** is built on top of the data frame and includes additional information on what the data frame holds. 
+
+## Data frames overview  
+
+A data frame is a data structure that consolidates the query results from your data sources, providing a common container in Grafana. 
+
+:::caution
+
+Query responses are often more than one single data frame. 
+
+:::
+
+The data frame a column-oriented table (the _fields_) with metadata (the _frame_) attached. Since data frame columns can have labels attached to them (`key=value`, `key2=val`...), it can hold Prometheus like responses as well. 
```

Comment:
```suggestion
The data frame is a column-oriented table (the _fields_) with metadata (the _frame_) attached. Since data frame columns can have labels attached to them (`key=value`, `key2=val`...), it can hold Prometheus like responses as well. 
```

I don't really understand what we mean by "it can hold Prometheus like responses as well"

Reply:
Yes, something like "responses that use labels, for example prometheus and other time series databases"